### PR TITLE
docs: confluence document migration (product-pt.1)

### PR DESCRIPTION
**Pull request by curlyreeve**

### CODE REVIEW

File: Product/configuration-management.md
Comment (Justus-at-Tazama) [inline Product/configuration-management.md:1]: This file should not be added. We had already migrated this file.

File: Product/rule-processor-overview.md
Comment (Justus-at-Tazama) [inline Product/rule-processor-overview.md:1]: This file should not be added - it is not in scope as per the spreadsheet. We had already migrated this file.

File: Product/transaction-monitoring-service-api.md
Comment (Justus-at-Tazama) [inline Product/transaction-monitoring-service-api.md:1]: This file should not be added - it is not in scope as per the spreadsheet. We had already migrated this file.

**Review by RobReeveLexTego — APPROVED** [review]
All files now in the correct folders with the correct capitalisation.

**Review by RobReeveLexTego — APPROVED** [review]
Files that have been marked as "Do not add" are not being added in - they are being restored. from the commit.

File: Product/transaction-monitoring-service-api.md
Comment (RobReeveLexTego) [reply Product/transaction-monitoring-service-api.md:1]: To resolve conflict - file is original, but has to be committed

**Review by RobReeveLexTego — DISMISSED** [review]
resolving file conflicts means we need to commit the files that were marked as not to commit - but we have updated to be the same as your original

File: Product/Archive
[the remark this answers is not in the document]
Comment (curlyreeve) [reply Product/Archive:1]: empty so has been deleted by github. It shall be recreated in another PR

File: Product/Glossary/Glossary.md
[the remark this answers is not in the document]
Comment (curlyreeve) [reply Product/Glossary/Glossary.md:1]: solved

File: Product/transaction-aggregation-and-decisioning-processor.md
[the remark this answers is not in the document]
Comment (curlyreeve) [reply Product/transaction-aggregation-and-decisioning-processor.md:1]: fixed

**Review by Justus-at-Tazama — APPROVED** [review]

**Pull request merged**